ingestion_tools/scripts/common/finders.py

```diff
@@ -249,22 +302,25 @@ class MultiSourceFileFinder(DefaultImporterFactory):
 
     def _get_instantiated_results(
         self,
-        cls,
         config: DepositionImportConfig,
         metadata: dict[str, Any],
         filtered_results: dict[str, str],
+        allow_imports: bool,
         parents: dict[str, Any] | None,
     ):
         if not filtered_results:
             return []
+        if any(path and path.endswith("metadata.json") for path in filtered_results.values()):
+            name, path, filtered_results = self.importer_cls.get_name_and_path(metadata, None, None, filtered_results)
```

Comment:
`get_name_and_path` is only implemented for the TiltSeriesImporter - are we never going to wind up in this situation for any other importers?

Reply:
This is going to be implemented only for alignment and tomogram for now. This will be added in a future PR, where those relationships are added.